app/views/user_mailer/creatorship_notification_archivist.html.erb

```diff
@@ -11,5 +12,7 @@
 
   <p><%= t(".explanation") %></p>
 
-  <p><%= t(".html.remove", edit: style_link(t(".html.edit", type: creation_type), edit_polymorphic_url(@creation))).html_safe %></p>
+  <p>
+    <%= t(".html.remove_#{creation_type}", edit: style_link(t(".html.edit_#{creation_type}"), edit_polymorphic_url(@creation))).html_safe %>
```

Comment:
Can you change `edit:` to `"edit_#{creation_type}_link":` here as well?

Reply:
Done